deploy/crds/openebs_v1alpha1_devicerequest_crd.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: deviceclaims.openebs.io
```

Comment:
Should this names be updated -- from `claims` to `request` ?

Reply:
Done.